struct-embedding/struct-embedding.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import "fmt"
+
+// Go supports embedding of structs and interfaces to express a more
+// seamless composition of types.
+type base struct {
+	num int
+}
+
+func (b base) describe() string {
```

Comment:
What happens if you implement a different `describe() string` method on `container` as well?